app/helpers/tasks_helper.rb

```diff
@@ -0,0 +1,42 @@
+module TasksHelper
+  TASKS_LIMIT = 5
+
+  def assigned_cards(project_id = nil)
+    @assigned_cards ||= begin
+      # Using Arel.sql to sort the records by due_date with null due_date records last
```

Comment:
I'd assume due_date null would be grouped at the beginning or the end of the regular `order(:due_date)`, no? Then it's a matter of choosing :desc / :asc and you get the same behavior w/ less code?